starter-code/styles/modules.css

```diff
@@ -13,6 +13,10 @@
   /* TODO: Add styles for the icon-menu class (whether it be the
       hamburger font icon or an image of your choosing). It should be
       bigger, and off to the right. */
+      float: right;
+      transform: scale(3);
```

Comment:
Unique solution! Since the menu is an icon font, another possibility can be to use the font-size property.